CustomResourceDefinition.yaml

```diff
@@ -0,0 +1,69 @@
+kind: List
```

Comment:
This is just for the tests, right?

I think this should be in a subdirectory so it's out of the way.

e.g. `test/crds.yaml`

(and a test/README.md)